tests/AdagucTests/TestWMS.py

```diff
@@ -1569,7 +1569,7 @@ def test_WMSGetMapQuantizeHigh(self):
 
         filename = "testWMSGetCapabilities_quantizehigh.xml"
         status, data, headers = AdagucTestTools().runADAGUCServer(
-            "DATASET=adaguc.tests.quantizelow&SERVICE=WMS&VERSION=1.3.0&REQUEST=GetCapabilities",
+            "DATASET=adaguc.tests.quantizehigh&SERVICE=WMS&VERSION=1.3.0&REQUEST=GetCapabilities",
```

Comment:
Where is the quantizelow test?

Reply:
That is in test_WMSGetMapQuantizeLow.
The test_WMSGetMapQuantizeHigh test erroneously used quantizelow configuration